src/librustc_errors/lib.rs

```diff
@@ -503,11 +519,18 @@ impl Handler {
     }
     pub fn delay_span_bug<S: Into<MultiSpan>>(&self, sp: S, msg: &str) {
         if self.flags.treat_err_as_bug {
+            // FIXME: don't abort here if report_delayed_bugs is off
```

Comment:
How straightforward is this to address in this PR?

Reply:
Oh it's easy. I just didn't want to change the default behaviour in this PR. ppl might get confused and not know about the extra flag you need to get the old behaviour back